lib/drab/live.ex

```diff
@@ -373,6 +373,18 @@ defmodule Drab.Live do
     end
   end
 
+  def socket_to_assigns(socket) do
```

Comment:
This will be a public API function, so it needs a documentation. Also, the name could be better, what about just `assigns`? (in this case it should rename existing private function to something more meaningful, like `assigns_values` or `assigns_for_partial` - as it returns a map with assigns and values for a specific partial).

To be consistent with the API the function should be like below:

    def assigns(socket) - returns assigns for the index partial (take it from index/1)
    def assigns(socket, partial) - where partial is "show.html" etc
    def assigns(socket, view, partial) - for a specific view (atom)

The existing private `assigns/3` takes partial hash (internal) and the partial name (the exact name, "show.html", used only in exception description). For the new `assigns/3`, you can take the partial hash from `partial_hash(view, partial_name)`. For `assigns/2` and `assigns/1`, you will need a view name for the partial (or for the main partial) - grab it from `Drab.get_view(socket)`